config/config.go

```diff
@@ -779,6 +780,47 @@ func getTxFeeConfig(v *viper.Viper, networkID uint32) fee.StaticConfig {
 	return genesis.GetTxFeeConfig(networkID)
 }
 
+func getUpgradeConfig(v *viper.Viper, networkID uint32) (upgrade.Config, error) {
```

Comment:
Decided this was better implemented here than broken into the `upgrade` package like for genesis.